src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
The names of an enum type's fields must be in uppercase letters because they are constants.
https://docs.oracle.com/javase/tutorial/java/javaOO/enum.html